internal/controllers/suite_test.go

```diff
@@ -74,13 +74,17 @@ func newClient(t *testing.T) client.Client {
 
 type MockInstalledBundleGetter struct {
 	bundle *controllers.InstalledBundle
+	err    error
 }
 
 func (m *MockInstalledBundleGetter) SetBundle(bundle *controllers.InstalledBundle) {
 	m.bundle = bundle
 }
 
 func (m *MockInstalledBundleGetter) GetInstalledBundle(ctx context.Context, ext *ocv1.ClusterExtension) (*controllers.InstalledBundle, error) {
+	if m.err != nil {
+		return nil, m.err
+	}
 	return m.bundle, nil
```

Comment:
nit: `return m.bundle, err` should be enough

Reply:
fixed